src/Native/LibMFWrapper/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+project (FactorizationMachineNative)
+
+set(SOURCES
+    UnmanagedMemory.cpp
+    mf.cpp
+)
+
+if(WIN32)
+else()
+    list(APPEND SOURCES ${VERSION_FILE_PATH})
+endif()
+
+add_library(LibMFWrapper SHARED ${SOURCES} ${RESOURCES})
+
+install_library_and_symbols (LibMFWrapper)
```

Comment:
>LibMFWrapper [](start = 29, length = 12)

We tend to call this library SomethingNative rather than SomethingWrapper.

Reply:
Just changed it to LIBMFNative.

---
In reply to: [225361005](https://github.com/dotnet/machinelearning/pull/1263#discussion_r225361005) [](ancestors = 225361005)